api/src/main/java/com/exelaration/abstractmemery/services/MetadataService.java

```diff
@@ -0,0 +1,21 @@
+package com.exelaration.abstractmemery.services;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.exelaration.abstractmemery.domains.Image;
+
+import org.springframework.stereotype.Service;
+
+public interface MetadataService {
+
+    public static final String uploadingDir = "/app/src/main/resources/images/";
```

Comment:
This exists in the FileStorageService and doesn't really seem to belong here.  Should this be removed?

Reply:
I will check and remove it if it can be.